src/views/Pools/components/PoolCard/AprRow.tsx

```diff
@@ -82,7 +83,7 @@ const AprRow: React.FC<AprRowProps> = ({ pool, isAutoVault = false, compoundFreq
 
   return (
     <Flex alignItems="center" justifyContent="space-between">
-      {tooltipVisible && tooltip}
+      {tooltipVisible && ReactDOM.createPortal(tooltip, document.getElementById('root'))}
```

Comment:
The placement is wrong, needs to be bottom-start. Originally it looked ok since tooltip readjusted itself
![Uploading Screenshot 2021-05-21 at 13.46.38.png…]()

Reply:
done